forum/templatetags/filter_img.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from BeautifulSoup import BeautifulSoup, Tag
+from os.path import basename, splitext
+
+register = template.Library()
+
+
+@register.filter(is_safe=True)
+def replace_img(string):
+    soup = BeautifulSoup(string)
```

Comment:
When I first thought of this, I considered just using string matching instead of parsing the html. Is this a viable solution? In hindsight I guess bs is a good solution for this, but we process html at submission time anyway, to make it only use a whitelist (https://github.com/MTG/freesound/blob/144a536168bf8763a2ab401d30e62775fc84eec4/utils/text.py#L209-L211).
Do we know what the processing time impact is of computing this each time a post is displayed?

Another thing to consider is that bs3 is no longer maintained. As far as I understand, bs3 and bs4 can be installed side-by-side. Should we consider slowly migrating new functionality to bs4?